ncm-spma/src/main/pan/components/spma/yum/schema.pan

```diff
@@ -22,9 +22,15 @@ type spma_yum_plugin_versionlock = {
     'follow_obsoletes' ? boolean
 };
 
+type spma_yum_plugin_priorities = {
+    'enabled' : boolean = true
```

Comment:
`ncm-spma` has a dependency on `yum-priorities` package, which ships a conf file that enables the plugin, so i'd say this is backwards compatible.